pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -529,7 +562,7 @@ in buildPythonPackage {
   # TEST_PACKAGES in tensorflow/tools/pip_package/setup.py
   nativeCheckInputs = [
     dill
-    keras
+    # keras
```

Comment:
```suggestion
```

Reply:
Makes sense, as long as everyone agrees that we don't want to pull in `keras` as a dependency again (see also https://github.com/NixOS/nixpkgs/issues/255733).

Reply:
Yeah, imho `propagatedBuildInputs` should contain only and exactly what is necessary to install the package